CMSgov/mint-app
# [NOREF] Adding id to queries and types

# NOREF

Addressing cache console errors mentioned [here](https://github.com/CMSgov/mint-app/pull/217#discussion_r974279352)
## Changes and Description

- Adds id to the queries 




## How to test this change

1. Navigate to the Model Summary ( Models on nav > Complete Plan > View this Model Plan on sidebar)
2. Open Developer tools and look at Console
3. Ensure there is no errors



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
